packages/@cdktf/hcl2cdk/test/tfExpressions.test.ts

```diff
@@ -239,6 +239,25 @@ ITEM
     }
   );
 
+  testCase.test(
+    "escaping ${} in template strings works",
+    `
+    resource "aws_ssoadmin_instance_access_control_attributes" "example" {
+      attribute {
+        value {
+          source = ["$\${path:name.givenName}"]
```

Comment:
This is different from the other unit test, is this by design?

Reply:
Good catch! The other test was off 😅